server/src/schema/event/eventType.ts

```diff
@@ -212,6 +253,22 @@ export const eventType = builder.prismaObject('TumiEvent', {
         },
       }),
     }),
+    ratings: t.relation('registrations', {
```

Comment:
I think it is misleading that `ratings` would return registrations, maybe add an input to the registrations relation that allows to specify `registrations(onlyWithRating: true)`